compiler/optimizer/LocalAnalysis.cpp

```diff
@@ -714,7 +714,12 @@ bool TR_LocalAnalysisInfo::isCallLike(TR::Node *node) {
 
    if (node->getOpCode().hasSymbolReference())
       {
-      if (node->getSymbolReference()->getSymbol()->isVolatile() || node->getSymbolReference()->getSymbol()->isMethodMetaData())
+      if (node->getSymbolReference()->getSymbol()->isVolatile() || 
+          (node->getSymbolReference()->getSymbol()->isMethodMetaData()
+#if J9_PROJECT_SPECIFIC
```

Comment:
I agree, but FrontEnd is not the right place for it as we have deprecated that data structure in OMR.  SymbolReference itself is an extensible class so you can define that question in the OMR layer (returning false, let's say) and then specialize it in a trj9 project to answer the question the way trj9 wants.